app/recipe/tests/test_ingredients_api.py

```diff
@@ -0,0 +1,70 @@
+from django.urls import reverse
+from django.test import TestCase
+
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from core.models import Ingredient, Recipe
+
+from recipe.serializers import IngredientSerializer
+
+
+INGREDIENTS_URL = reverse('recipe:ingredient-list')
+
+
+class IngredientsApiTests(TestCase):
+    """Tests ingredients api"""
+
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_retrieve_ingredient_list(self):
+        """Test retrieving a list of the ingredients"""
+        Ingredient.objects.create(name='Kale')
+        Ingredient.objects.create(name='Salt')
+
+        res = self.client.get(INGREDIENTS_URL)
+
+        ingredients = Ingredient.objects.all()
+        serializer = IngredientSerializer(ingredients, many=True)
```

Comment:
It would be better to explicitly check for the data returned, in order to decouple the test from the serializer.